strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -566,12 +569,19 @@ def prepare_gkp(self, state, epsilon, ampl_cutoff, representation="real", shape=
         if representation == "complex":
             raise NotImplementedError("The complex description of GKP is not implemented")
 
-        if shape != "square":
-            raise NotImplementedError("Only square GKP are implemented for now")
+        if shape not in ["square", "rectangular"]:
+            raise NotImplementedError("Only square and rectangular GKP are implemented.")
+
+        if shape == "square":
+            if alpha != 1:
+                raise ValueError(
+                    "For square GKPs, alpha must be 1. For alpha not equal to "
+                    + "1, use shape='rectangular'."
```

Comment:
```suggestion
                    + "1, use shape=\"rectangular\"."
```

Reply:
Funnily enough, we tend to use the convention of single quotation marks in error messages, partly because this is what the python standard library does!

Even though double quotations are recommended in code, single quotations seem to be the default approach when printing:
![image](https://user-images.githubusercontent.com/2959003/147046281-4233289b-81c5-4d5f-b70c-ade4c3e85156.png)